### PR TITLE
HAWQ-681. Removed hcatalog_enable GUC.

### PR DESCRIPTION
**Pull request by sansanichfb**

### CODE REVIEW

**Comment by kavinderd** [comment]
LGTM

File: src/test/regress/input/hcatalog_lookup.source [hunk 1]
```diff
@@ -2,16 +2,9 @@
 -- test hcatalog lookup
 -- --------------------------------------
 
--- Negative test with GUC disabled
-SET hcatalog_enable = false;
-SELECT * from hcatalog.db.t;
-
 SELECT * FROM pxf_get_item_fields('Hive', '*');
```
Comment (shivzone) [inline src/test/regress/input/hcatalog_lookup.source]: for each of the higher level use case. please add a one line comment on the type of test

**Comment by shivzone** [comment]
+1

File: src/test/regress/input/hcatalog_lookup.source (hunk 1 above)
Comment (sansanichfb) [inline src/test/regress/input/hcatalog_lookup.source]: @shivzone good to have, but not related to this JIRA

**Pull request closed without merging**